lib/transport-common.js

```diff
@@ -68,25 +68,23 @@ const newConnectAndInspectFn = (connFactory, transportClass) => {
     const callback = common.extractCallback(options);
     connect(appName, client, ...options, (error, connection) => {
       if (error) return callback(error);
-
-      Promise.all(interfaces.map(name => new Promise((resolve, reject) => {
-        connection.inspectInterface(name, (error, proxy) => {
-          if (error) {
-            reject(error);
-          } else {
-            resolve(proxy);
+      const proxies = {};
```

Comment:
This object should be created with `Object.create(null)` like it was done before this patch.